examples/Joints/Ingen/slidingjoint.py

```diff
@@ -0,0 +1,21 @@
+from sympy import Symbol
```

Comment:
What are you creating here? Is it a single mass on a spring that has gravity in the same direction as the mass degree of freedom?